### PR TITLE
Add Edit Activity Page

### PR DESCRIPTION
**Pull request by Christinarlong**
**Summary:**
- Added ability to edit activities by clicking on activity cards in submissions page
- Added Edit page route
- Modified FormInput to have an editing and non editing state
- Added ErrorBanner component (see image)
- Added last date modified label

**To run**
- Pull branch and click on any card in the submissions page
- Not exactly sure how to trigger an error naturally right now, maybe disconnect your db or smthn. Or change `toggleEditingError` to be true even on `ResponseStatus.Success` in `FormInput.tsx` line 272

**Images:**
image
image

### CODE REVIEW

File: src/components/ActivityForm/FormInput.tsx [hunk 1]
```diff
@@ -409,7 +529,12 @@ const FormInput: React.FC = () => {
       </div>
 
       <div className="flex justify-between items-center cursor-pointer my-9">
-        <button onClick={() => dispatch(setStep('selection'))}>Back</button>
+        {isEditing ? (
```
Comment (maxpinheiro) [inline src/components/ActivityForm/FormInput.tsx:532]: we can do this with a single button, and put the `isEditing?` logic inside the `onClick`:

```
<button onClick={() => isEditing ? router.back() : dispatch(setStep('selection'))}>Back</button>
```

**Review by maxpinheiro — CHANGES_REQUESTED** [review]
Logic looks good, just a couple small changes. There are also conflicts with master, so please rebase your branch onto master, and then make the changes.

File: src/components/Narratives/NarrativeInstructions 2.tsx
Comment (maxpinheiro) [inline src/components/Narratives/NarrativeInstructions 2.tsx:1]: looks like a bunch of duplicate files got added during a rebase (`NarrativeInstructions 2.tsx`, `[...nextauth] 2.ts`, `profile.store 2.ts`, etc. the duplicates are identical to the original files, so let's just delete all the duplicates

**Review by maxpinheiro — CHANGES_REQUESTED** [review]
Looks good, let's just clean up the duplicate files and then should be good to go

**Review by maxpinheiro — APPROVED** [review]
Looks great! 🚢

**Comment by Christinarlong** [comment]
> Looks great! 🚢

thanks sexy

**Pull request merged**